src/key-value-pairs/internal.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.css.js';
+import { KeyValuePairsProps } from './interfaces';
+import { InfoLinkLabelContext } from '../internal/context/info-link-label-context';
+import ColumnLayout from '../column-layout/internal';
+import Box from '../box/internal';
+
+export { KeyValuePairsProps };
```

Comment:
You already are exporting this in index file, you don't need to export once again

Reply:
updated